code/src/InputName.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+const InputName = ({ question, name, setName }) => {
+  
+  return (
+    <div className="input-name"> 
+      <label 
+        className="input-text-label" 
+        htmlFor="InputText"> 
+        <h3 tabIndex="0">{question}</h3> 
+        <input
+          className="input-text"
+          id="InputText"
+          placeholder="Type your name"
+          type="text"
+          value={name}
+          required //This don't work. I don't know why.
+          minLength="2" //This don't work. I don't know why.
```

Comment:
Yep, in controlled inputs we need to check max and min value in JS, probably good thing to do it would be inside `setName` function :

```
const onNameChange = name => {
    if (name.length >= 2) {
        setName(name);
    }
};
```